src/constants/routes.ts

```diff
@@ -0,0 +1,9 @@
+const routes = {
```

Comment:
Why don't you use enum here? Surely you can use object. But I think enum is more logical.